habitat-baselines/habitat_baselines/rl/ppo/habitat_evaluator.py

```diff
@@ -89,7 +89,7 @@ def evaluate_agent(
             rgb_frames: List[List[np.ndarray]] = [
                 [
                     observations_to_image(
-                        {k: v[env_idx] for k, v in batch.items()}, {}
+                        {k: v[env_idx] for k, v in batch.items()}, {}, index=0
```

Comment:
what does this index 0 stand for?

Reply:
The index represents frame index for image frame generation. I will change the parameter name to `frame_id`